### PR TITLE
[175846490] story crud actions, story policy and migration

### PR DESCRIPTION
**Pull request by JelenaM-Kolosek**

### CODE REVIEW

File: app/models/story.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class Story < ApplicationRecord
+  belongs_to :user
+
+  scope :stories_desc, -> { order(created_at: :desc) }
+  scope :stories_desc_author, ->(user) { where(user_id: user.id).order(created_at: :desc) }
```
Comment (tamara-bol) [inline app/models/story.rb]: No need to combine scopes - one scope should do one thing. You can latter concatenate them when doing the call. For example:

@stories.stories_desc().stories_author()

File: app/models/user.rb [hunk 2]
```diff
@@ -14,6 +16,8 @@ def set_admin
   end
 
   scope :pending, -> { where.not(invitation_token: true) }
+  # User
+  scope :accepted, -> { where(!User.where.not(invitation_token: true)) }
```
Comment (tamara-bol) [inline app/models/user.rb]: !User.where.not(invitation_token: true) - double negation, it is not readable

also how does this scope work? why there is a need for nesting, when you can just directly check the invitation_token property?
  scope :accepted, -> { where.not(invitation_token: true)} - isnt that enough?

File: app/policies/story_policy.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+class StoryPolicy < ApplicationPolicy
+  class Scope < Scope
+    def resolve
+      if user.admin? || user.editor?
+        scope.stories_desc
```
Comment (tamara-bol) [inline app/policies/story_policy.rb]: consider to use default scope for ordering, if you need the order every time

**Review by tamara-bol — CHANGES_REQUESTED** [review]

File: app/models/user.rb (hunk 2 above)
Comment (JelenaM-Kolosek) [reply app/models/user.rb]: I wanted users who have accepted the invitation. I changed it now in members overview branch to "scope :accepted, -> { where.not(sign_in_count: 0) }".

**Review by tamara-bol — APPROVED** [review]

**Pull request merged**